src/lib/components/Modal/Modal.jsx

```diff
@@ -184,6 +253,10 @@ Modal.propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     current: PropTypes.any,
   }),
+  /**
+   * Enforces focus to stay inside of Modal. Works only if autoFocus is not disabled.
+   */
+  focusEnforce: PropTypes.bool,
```

Comment:
Since I think that focus trap should be enabled by default, I'd prefer disableFocusTrap. Option to disable it should take some action for users that know what they are doing.

Reply:
We do not use imperative verbs for boolean props. We use `raised`, `muted`, `disabled` etc. (adjectives I gues).

However I would prefer naming the prop `focusTrap` as that in my view is descriptive enough. We can set it default true, that should cause no confusion.

@hubacekj I see where you are coming from, but we try to stay away from names in negative. It makes the code harder to read when you have double and triple negatives such as `if (!disableFocusTrap)…`. It is a convention we maintain across code bases (I think at least).

Reply:
> enabled by default

… and that's just fine. We set it to `true` by default and keep the name.

Reply:
Also, from user's point of view, the API would soon become confusing if there was a mix of `enable*` and `disable*` options. My personal opinion is that authors of such APIs just react on the requests coming (like "we have it there but now we need to turn it off somewhere") and don't think much about usability in the downstream. Having the API consistent no matter the default value helps users (= library consumers) adopt and use the components.